msteams-platform/index.yml

```
@@ -14,100 +15,120 @@ metadata:
         - url: /learn/modules/embedded-web-experiences
           itemType: learn
-          text: Tabs — create embedded web experiences 
+          text: Create embedded web experiences with tabs
         - url:  /learn/modules/msteams-conversation-bots/
           itemType: learn
-          text: Bots — create conversational interactions
+          text: Create conversational interactions with bots
         - url: /learn/modules/msteams-messaging-extensions/
           itemType: learn
-          text: Messaging extensions — create task-oriented interactions 
+          text: Create task-oriented interactions with messaging extensions
         - url: /learn/modules/msteams-task-modules/
           itemType: learn
-          text: Task modules — collect input and gather information
+          text: Collect input and gather information with task modules
         - url: /learn/modules/msteams-webhooks-connectors/
           itemType: learn
-          text: Webhooks and Office 365 connectors — connect web services to channels and teams
+          text: Connect web services to channels and teams with webhooks and Office 365 connectors
     # Card
-    - title: Develop
-      links:
-        - url: /microsoftteams/platform/tutorials/get-started-nodejs-app-studio
-          itemType: get-started
-          text: Build a Teams app using JavaScript/Node.js and App Studio
-        - url: /microsoftteams/platform/tutorials/get-started-dotnet-app-studio
-          itemType: get-started
-          text: Build a Teams app using C#/.NET and App Studio
-     # Card
-    - title: Deploy
+    - title: Integrate
       links:
+        - url:  /microsoftteams/platform/samples/integrating-web-apps
+          itemType: reference
+          text: Integrate web apps with Microsoft Teams
+        - url: /microsoftteams/platform/bots/how-to/add-power-virtual-agents-bot-to-teams
+          itemType: reference
+          text: Integrate your Power Virtual Agents bot with Microsoft Teams
+        - url: /microsoftteams/platform/tabs/how-to/tabs-in-sharepoint
+          itemType: reference
+          text: Integrate Microsoft Teams and SharePoint 
+        - url: /microsoftteams/platform/resources/moodleinstructions
+          itemType: reference
+          text: Integrate Microsoft Teams and Moodle    
         - url: /microsoftteams/platform/samples/app-templates
-          itemType: deploy
+          itemType: reference
           text: Teams app templates — production-ready applications
         - url:  /microsoftteams/platform/samples/shifts-wfm-connectors
-          itemType: deploy
-          text: Teams shifts connectors - production-ready integrations
-        - url: /microsoftteams/platform/bots/how-to/add-power-virtual-agents-bot-to-teams
-          itemType: deploy
-          text: Integrate your Power Virtual Agents bot with Teams     
+          itemType: reference
+          text: Teams shifts connectors — production-ready integrations
```

```suggestion
          text: Integrate production-ready Shifts connectors
```
Suggested change for parallel structure. "Shifts" is capitalized.